### ISSUE TITLE
Full profile name in file names

### ISSUE THREAD

**Issue by cschleter** [HuiiBuh/InstagramDownloader#75]
It would be really helpful if the generated filename for downloads included the full profile name (including special characters) before the GUID. eg: _cs_utah instead of csutah.

**Comment by HuiiBuh** [comment]
I agree. This will be fixed soon

**Comment by cschleter** [comment]
Great. Thanks!

While you're at it, what would be the chance that you could add a feature 
to let the user select a default save directory?

**Comment by HuiiBuh** [comment]
If you mean android I am not even sure if that is possible...

**Comment by HuiiBuh** [comment]
Just uploaded the new version

**Comment by cschleter** [comment]
Not Android. I was talking about Windows.

**Comment by cschleter** [comment]
I'll take a look when I get home this afternoon.

**Comment by cschleter** [comment]
Finally got the update. Thanks for making the change.

**Comment by HuiiBuh** [comment]
! [Image](https://i.imgur.com/LIpBhjGr.png) Is this what you had in mind with the location?

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/downloads/download

**Comment by cschleter** [comment]
Not really and I think that the issue actually be with the 
configurations of my laptop and desktop. On my desktop, Firefox 
remembers the folder I saved to the last time I saved a file from a 
website and defaults to that folder each time I save something. My 
laptop defaults to the Downloads folder. It appears that neither of the 
defaults can be changed. It's probably something they changed in Firefox 
that nobody was aware of and it isn't really a big deal. It would just 
be convenient to be able to change it.

Thanks
